server/server.js

```diff
@@ -0,0 +1,25 @@
+const express = require('express');
+const app = express();
+const port = 3000;
+const bodyParser = require('body-parser');
+// const database = require('./database/database.js');
+
+
+app.use(express.static(__dirname + '/../dist'));
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(bodyParser.json());
+
+app.get('/api/homes', (req, res) => {
```

Comment:
you could split up the server side into models and controllers where the controllers handle the get/post requests and the models handle querying the database. not required, but similar to what we've seen in some past sprints.

Reply:
i'll implement these changes as well--better to have a more professionally organized repository